### DISCUSSION TITLE
add packages permissions for other repos via api calls

### DISCUSSION THREAD

**Discussion by Ron-Be** [community/community#144420] [Apps, API and Webhooks]
### Select Topic Area

General

As I understand it, currently the only way to grant GitHub Actions access to packages is through the [GitHub UI](https://docs.github.com/en/packages/learn-github-packages/configuring-a-packages-access-control-and-visibility#github-actions-access-for-packages-scoped-to-personal-accounts). We would like the option to use an API call to add new repositories to all of our packages, streamlining the process and reducing reliance on manual updates in the UI

Edit:
As a workaround Iv'e changed my package visibility to be internal which made it visible to my entire org, not quite the solution I was hoping for but it will do for now.

**Comment by smokedlinq** [comment]
Definitely something that's needed. We have a repo with several dozen packages it publishes and would like to be able to grant all the org repos access to them either with a setting or via automation so we don't have to share a PAT with the org.

**Reply by Ron-Be** [reply]
"To enable organization members to create internal packages that are visible to all organization members, click Internal. If the organization belongs to an enterprise, the packages will be visible to all enterprise members" 

In the package danger zone you can change the current visibility, hope this help !
